changelog/@unreleased/pr-2279.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: feature
+feature:
+  description: |-
+    Add support for routing through HTTPS proxies in the apache client when configured
+    <!-- User-facing outcomes this PR delivers -->
```

Comment:
nit: delete this line

Reply:
Deleted, does changelog bot work differently here 🤔